src/bp-core/admin/settings/bp-admin-setting-activity.php

```diff
@@ -150,24 +134,48 @@ public function register_fields() {
 				'description' => false,
 			);
 
+			$post_type_option_name = bb_post_type_feed_option_name( $post_type );
+			$comment_option_name   = bb_post_type_feed_comment_option_name( $post_type );
+
 			if ( 'post' === $post_type ) {
+				$parent_fields = array_merge(
+					$fields['args'],
+					array( 'class' => 'child-no-padding-first' )
+				);
+
+				$fields['args']['class'] = 'child-no-padding bp-display-none';
+
 				// create field for each of custom post type.
-				$this->add_field( "bp-feed-custom-post-type-$post_type", __( 'WordPress', 'buddyboss' ), 'bp_feed_settings_callback_post_type', 'intval', $fields['args'] );
+				$this->add_field( $post_type_option_name, __( 'WordPress', 'buddyboss' ), 'bp_feed_settings_callback_post_type', 'intval', $parent_fields );
 				// Activity commenting on post and comments.
-				$this->add_field( 'bp-disable-blogforum-comments', __( 'Post Type Comments', 'buddyboss' ), 'bp_admin_setting_callback_blogforum_comments', 'bp_admin_sanitize_callback_blogforum_comments' );
+				$this->add_field( $comment_option_name, '&#65279;', 'bb_feed_settings_callback_post_type_comments', 'intval', $fields['args'] );
+
```

Comment:
@asaquzzamanbb check and update the code as below.
```
// create field for each of custom post type.
$this->add_field( $post_type_option_name, __( 'WordPress', 'buddyboss' ), 'bp_feed_settings_callback_post_type', 'intval', $fields['args'] );

$fields['args']['class'] = 'child-no-padding bp-display-none';

// Activity commenting on post and comments.
$this->add_field( $comment_option_name, '&#65279;', 'bb_feed_settings_callback_post_type_comments', 'intval', $fields['args'] );
```

Reply:
Done!!

https://github.com/buddyboss/buddyboss-platform/pull/2290/commits/6007401be709c486f34f833cfe8002a5cdc340ee